draft-ietf-rats-corim.md

```diff
@@ -2417,6 +2502,27 @@ If key verification succeeds for any _key_:
 
 Otherwise, do not add the `addition` ECT to the ACS.
 
+#### Processing Domain Membership {#sec-process-dm}
```

Comment:
This can be improved, but it’s good enough for now.  Thanks for addressing my comment.